packages/addons/tools/jre.zulu/package.mk

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2019-present Peter Vicman (peter.vicman@gmail.com)
+
+PKG_NAME="jre.zulu"
+PKG_VERSION="1.0"
+PKG_REV="101"
+PKG_LICENSE="GPL2"
+PKG_DEPENDS_TARGET="jdk-${TARGET_ARCH}-zulu jre-libbluray libXext chrome-libXtst chrome-libXi chrome-libXrender jre-libXinerama"
```

Comment:
```suggestion
PKG_DEPENDS_TARGET="jdk-${TARGET_ARCH}-zulu jre-libbluray libXext chrome-libXtst chrome-libXi chrome-libXrender jre-libXinerama"
PKG_SECTION="tools"
```

Reply:
I think PKG_SECTION is usable only for addons so I would say it belongs to addons section but ok :)

Reply:
Yea this is pretty useless this days but we still need this just for Jenkins, taht get mass removed some day